infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/Names.java

```diff
@@ -20,40 +25,37 @@
  * @author Sergii Leshchenko
  */
 public class Names {
+
   static final char WORKSPACE_ID_PREFIX_SEPARATOR = '.';
 
   static final String ROUTE_PREFIX = "route";
   static final int ROUTE_PREFIX_SIZE = 8;
 
-  /** Returns machine name that has the following format `{POD_NAME}/{CONTAINER_NAME}`. */
-  public static String machineName(String podName, String containerName) {
-    return podName + '/' + containerName;
-  }
-
-  /** Returns machine name that has the following format `{POD_NAME}/{CONTAINER_NAME}`. */
-  public static String machineName(Pod pod, Container container) {
-    return machineName(pod.getMetadata().getName(), container.getName());
-  }
-
   /**
-   * Returns pod name that is extracted from machine name.
-   *
-   * @see #machineName(String, String)
+   * Returns machine name that has the following format `{POD_NAME}/{CONTAINER_NAME}` or machine
```

Comment:
Please rewrite java doc here to make clear that there are three possible ways in which machine name can be constructed. 
I mean
1. MACHINE_NAME_ANNOTATION_FMT
2. CHE_ORIGINAL_NAME_LABEL
3. Names of objects